cmd/beacon-chain/flags/base.go

```diff
@@ -205,7 +205,12 @@ var (
 	// EnableRegistrationCache a temporary flag for enabling the validator registration cache instead of db.
 	EnableRegistrationCache = &cli.BoolFlag{
```

Comment:
Feature flags should not be in this package, its the reason we have a separate feature package for flags like this. It makes it easier to add/remove flags like this. This package should always have permanent flags